src/core/components/SlQueryBuilder/SlQueryBuilder.css

```diff
@@ -0,0 +1,45 @@
+@import url('@ni/nimble-tokens/dist/styledictionary/css/variables.css');
+
+body {
```

Comment:
There are a lot of duplicated constants and no comments about where thee values came from.

For the colors in particular can you make named aliases and comments to where these values are coming from?

```css
body {
   /* Tokens from: <src_url> */
    --ni-grafana-query-builder-blue: #3E3E42
}

/* ... */ {
    --smart-scroll-bar-background: var(--ni-grafana-query-builder-blue);
    --smart-scroll-bar-track-background:  var(--ni-grafana-query-builder-blue);
}
```